ingress/controllers/gce/README.md

```diff
@@ -381,22 +382,7 @@ metadata:
 type: Opaque
```

Comment:
Nit: This example yaml would read better if the data fields were at the end, and Kind, metadata.name fields were at the beginning.